source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/display.tsx

```diff
@@ -65,9 +65,19 @@ export default class DirectoryDisplay extends React.Component<IProps> {
             {t('directory.title', { numProjects: 0 })}
           </h2>
 
-          <DebouncedSearch
-            placeholder={t('common.search')}
-            onSubmit={this.search} />
+          <Popup
+            basic
+            hoverable
+            trigger={<div>
+              <DebouncedSearch
+                placeholder={t('common.search')}
+                onSubmit={this.search} />
+            </div>}
+            position='bottom center'>
+
+            <div dangerouslySetInnerHTML={{ __html: t('directory.search-help') }} />
```

Comment:
couldn't you just use "content property" of Popup?

I know you did it to add the <br> into it (but IMHO: HTML shouldn't be part of the translation file)

Reply:
see top-level comment

Reply:
can do simple <li> html? if not go ahead and merge it

Reply:
That'd be a lot more markup